.github/workflows/ci.yml

```diff
@@ -20,16 +20,20 @@ jobs:
 
         python-version:
           - 3.9
+        llvm-version:
+          - "10.0"
+        include:
+          - { os: windows-latest, dc: ldc-latest, python-version: 3.9, llvm-version: "16.0" }
```

Comment:
Window is here but not in the os list? Also, please don't use `latest` for anything since that means the build isn't reproducible.

Reply:
since windows dmd isn't supported yet, and all the compilers are only dmd right now, I added a separate "include" list, where windows is only listed with ldc (a single entry right now)